react/components/ProductImages/components/Carousel/index.js

```diff
@@ -268,13 +307,24 @@ class Carousel extends Component {
           </Swiper>
         </div>
         <div className={imageClasses}>
-          <Swiper {...galleryParams} ref={this.gallerySwiper}>
+          <Swiper {...this.galleryParams} ref={this.gallerySwiper}>
             {slides.map((slide, i) => (
               <div key={i} className="swiper-slide center-all">
                 {this.renderSlide(slide, i)}
               </div>
             ))}
           </Swiper>
+          {zoomType === 'gallery' && (
+            <NoSSR>
+              <Gallery
+                items={slides}
+                index={selectedIndex}
+                isOpen={isGalleryOpen}
+                handleClose={() => this.setState({ isGalleryOpen: false })}
```

Comment:
create a function and pass the handle to this function like `handleClose={this.handleGalleryClose}`